src/commcare_cloud/ansible/wipe_kafka.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Delete all Kafka topics
+  become: true
+  hosts: zookeeper  # Doesn't need to run on all Kafka hosts
+  tasks:
+    - name: Check wipe_environment_enabled has been set to True
+      assert:
+        that: "{{ wipe_environment_enabled|default(False) }} == True"
+        fail_msg: 'This playbook will delete all data. To continue, set
+          "wipe_environment_enabled: True" in public.yml. Take care to unset
+          "wipe_environment_enabled" and run the `deploy_kafka.yml` playbook
+          again when the environment setup is complete.'
```

Comment:
as an extra precaution you could add a prompt to confirm: https://docs.ansible.com/ansible/latest/user_guide/playbooks_prompts.html